content/200-concepts/100-components/01-prisma-schema/10-postgresql-extensions.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: 'PostgreSQL extensions'
+metaDescription: 'How to represent PostgreSQL extensions in your Prisma schema'
+hidePage: false
+preview: true
+tocDepth: 2
+---
+
+<TopBlock>
+
+This page introduces PostgreSQL extensions and describes how to represent extensions in your Prisma schema, how to introspect existing extensions in your database, and how to apply changes to your extensions to your database with Prisma Migrate.
+
+<Admonition type="warning">
+
+Support for declaring PostgreSQL extensions in your schema is available in preview for the PostgreSQL connector only in Prisma versions 4.5.0 and later.
+
+</Admonition>
+
+</TopBlock>
+
+## What are PostgreSQL extensions?
+
+PostgreSQL allows you to extend your database functionality by installing packages known as _extensions_. For example, the `citext` extension adds a case-insensitive string data type.
+
+To install an extension, it must already be available on the local file system of the your database server. Installing it then runs a script file that adds the new functionality.
+
+Prisma's `postgresExtensions` preview feature allows you to represent PostgreSQL extensions in your Prisma schema. Note that specific extensions may add functionality that is not currently supported by Prisma. For example, an extension may add a type or index that is not supported by Prisma. This functionality must be implemented on a case-by-case basis and is not provided by this preview feature.
+
+## How to enable the <inlinecode>postgresExtensions</inlinecode> preview feature
+
+Representing PostgreSQL extensions in your Prisma schema is currently a preview feature. To enable the `postgresExtensions` preview feature, you will need to add the `postgresExtensions` feature flag to the `previewFeatures` field of the `generator` block in your Prisma schema file:
```

Comment:
Can we just add this whole section to the Admonition up top, about this being a preview feature, to keep all info about it being in preview together? Will make future maintenance (when it goes HGA) much easier.

Reply:
Possibly, but this is how we've been doing it so far, and I'm not sure this whole code block will display well in an Admonition